README.md

```diff
@@ -56,6 +56,17 @@ current_version = Stanford::StorageServices.current_version('oo000oo0000') # whe
 expect(current_version).to be_an_instance_of Integer
 ```
 
+#### Get Inventory From Content Metadata
+
+To generate a `Moab::FileInventory` object containing fixity, size, and other info:
+
+```ruby
+require 'moab/stanford'
+doc = IO.read('path/to/contentMetadata.xml')
+sci = Stanford::ContentInventory.new
+sci.inventory_from_cm(doc,'druid:th154ru1456','all','10') # all of v0010
+```
+
```

Comment:
3 things:

1.  is this stanford specific?  If not, move it above line 37

2.  it might be nice to show a snippet of example output, or some rspec-y stuff to show what the result will be

3.  spaces after commas, please

Reply:
agreed, and to add one more nitpick, i'd say that the `require 'moab/stanford'` seem unnecessary, just because it's mentioned first thing under "Usage", and because the other examples don't include it.